CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and **Merged pull requests**. Critical items to know are:
 The versions coincide with releases on pip. Only major versions will be released as tags on Github.
 
 ## [0.0.x](https://github.com/singularityhub/sregistry-cli/tree/master) (0.0.x)
+ - allow anonymously accessing S3 buckets (0.2.24)
+ - add sregistry.utils.confirm_action
```

Comment:
Nitpick - if this is under the same version, just indent the lines two spaces for the second bullet.

Reply:
Fixed

Reply:
Fixed properly now 😄